packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -147,3 +147,48 @@ test.describe('Filesharing ProgressBar', async () => {
     );
   });
 });
+
+test.describe('Filesharing SendBox Errorbar', async () => {
+  test.skip(isTestProfileStableFlavor());
+
+  test.beforeEach(async ({ pages, users, serverUrl }) => {
+    await chatTestSetup({ pages, users, serverUrl });
+  });
+
+  test('shows file too big error', async ({ serverUrl, users, page }) => {
```

Comment:
Let's make this test name generic.
Something like `shows file upload error`
Since this test case tests for the error bar UI and not the underlying implementation

Reply:
Done